.idea/codeStyles/Project.xml

```diff
@@ -1,8 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <AndroidXmlCodeStyleSettings>
-      <option name="ARRANGEMENT_SETTINGS_MIGRATED_TO_191" value="true" />
-    </AndroidXmlCodeStyleSettings>
```

Comment:
Was this deleted or eaten by rebase?

Reply:
uhh there were recently some gradle/androidstudio updates and personally i think those might have been the cause of this being changed or deleted?? when building the project after the updates? i'm not sure whether to keep it or not since i don't know what it exactly it does....

@felixlapierre could i get your input here?

Reply:
It's been cropping up on all the pull requests, don't worry about it. It's just a code style configuration.